examples/platform/esp32/common/Esp32AppServer.cpp

```diff
@@ -109,6 +114,9 @@ void Esp32AppServer::Init(AppDelegate * sAppDelegate)
 #if CHIP_DEVICE_CONFIG_ENABLE_WIFI
     sWiFiNetworkCommissioningInstance.Init();
 #endif
+#if CHIP_DEVICE_CONFIG_ENABLE_ETHERNET
```

Comment:
Are CHIP_DEVICE_CONFIG_ENABLE_WIFI and CHIP_DEVICE_CONFIG_ENABLE_ETHERNET mutually exclusive?  This code is assuming they are.  It should probably #error if both are defined.

Reply:
I have modified this change and made Wifi and Ethernet mutually exclusive